Add "view site" link to admin panel SN-440 (#16)
saritasa-nest/saritasa-forge-admin: README.md

You can sort multiple properties at once. It can be achieved by pressing sort buttons with `CTRL`.

Sorting can be cancelled by pressing on it with `ALT`.

## External link

On the top right side of the admin panel, there is a "View Site" link that can be configured to link to another website.
By default, the link will open the home page of the application.

### Using Fluent API

```csharp
services.AddNetForge(optionsBuilder =>
{

>     optionsBuilder.ConfigureUrl("https://stackoverflow.com/");

Reviewer: Can you replace it with www.example.com?

Sorting can be cancelled by pressing on it with `ALT`.

> ## External link

Reviewer: It's supposed to redirect to the website itself. I'm not sure we should call it External link. Actually, the admin panel is hosted within the main website so it's not an external one.

Call it `View Site URL`

> Located in the top right corner of the admin panel is a "View Site" link, configurable to direct users to an external website.

Reviewer: ```suggestion
Located in the top right corner of the admin panel is a "View Site" link, configurable to direct users to the website URL.
```

> The link supports two types of URLs: full paths, e.g., "https://www.example.com/", and relative routes, such as "/index".

Reviewer: ```suggestion
The default URL is "/".
```

> If left unconfigured, the link will default to opening the home page of the application.

Reviewer: ```suggestion
You can customize this value using the Fluent API:
```
